rust/utils/run-locally/src/invariants.rs

```diff
@@ -84,7 +84,7 @@ pub fn termination_invariants_met(
     )?
     .iter()
     .sum::<u32>();
-    if dispatched_messages_scraped != eth_messages_expected {
+    if dispatched_messages_scraped != eth_messages_expected + ZERO_MERKLE_INSERTION_KATHY_MESSAGES {
```

Comment:
let's create a ticket to track this

Reply:
created here: https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2819